src/goto-programs/remove_function_pointers.cpp

```diff
@@ -90,6 +107,12 @@ class remove_function_pointerst:public messaget
   }
 };
 
+static irep_idt get_member_identifier(const member_exprt &member)
+{
+  return id2string(member.symbol().get_identifier()) +
+      id2string(member.get_component_name());
```

Comment:
Suggest indexing this map by `exprt`s rather than strings, since it will surely be useful sooner or later to decompose them again